oep-template.rst

```diff
@@ -85,32 +77,20 @@ An OEP that introduces backward incompatibilities must describe the
 incompatibilities, with their severity and an explanation of how you propose to
 address these incompatibilities.
 
-OEP submissions that do not consider backward compatibility issues are unlikely
-to be accepted.
-
-
 Reference Implementation
 ========================
 
 The reference implementation must be completed before any OEP is given "Final"
-status, but it need not be completed before the OEP is accepted. While there is
+status, but it need not be completed before the OEP is "Accepted". While there is
 merit to the approach of reaching consensus on the specification and rationale
 before writing code, the principle of "rough consensus and running code" is
-still useful when it comes to resolving many discussions of API details.
-
-The final implementation must include test code and documentation, following
```

Comment:
Why remove this requirement?  Do you find that we are not actually holding ourselves to this standard?

Reply:
See my perspective above regarding the rationale for simplifying the process.

Given that, I find having instructions about the implementation to be orthogonal to the OEP-1 process.  I am thinking each OEP should be focused on its own goals, otherwise, they become too fat when they refer to other related aspects.